fixtures/ios_app_with_templates/Tuist/Templates/custom_using_attribute/custom_using_attribute.swift

```diff
@@ -0,0 +1,26 @@
+import ProjectDescription
+
+let supportingPlatforms: [String: Template.Attribute.Value] = [
+    "iOS": true,
+    "macOS": true,
+    "watchOS": false,
+]
+
+let nameAttribute: Template.Attribute = .required("name")
+let platformAttribute: Template.Attribute = .optional("platforms", default: .dictionary(supportingPlatforms))
```

Comment:
Thanks to `ExpressibleByDictionaryLiteral`, it is now possible to express `.optional("platforms", default: supportingPlatforms)`. I will make modifications to this part after the test is finished.